Sources/PointFree/RunnerTasks/WelcomeEmails.swift

```diff
@@ -12,49 +12,60 @@ import Prelude
 import Styleguide
 import Views
 
-public func sendWelcomeEmails() -> EitherIO<Error, Prelude.Unit> {
+public func sendWelcomeEmails() async throws {
```

Comment:
After getting a passing test I was finding that the complex `EitherIO`/`async` delay+retry hybrid logic when applied would occasionally double-increment the episode credit count. Rather than figure out why EitherIO was having this issue, I unwound the code to just be `async` and it fixed the test expectation.